x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -298,12 +298,20 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
           order: 9000,
           euiIconType: APP_ICON_SOLUTION,
         },
+        {
+          id: SecurityPageName.detections,
+          title: DETECTION_ENGINE,
+          path:'/detections',
+          navLinkStatus: AppNavLinkStatus.visible,
+          order: 9000,
```

Comment:
I don't know exactly why the `900*` but we better keep the same `order: 9001` as before